cuqi/experimental/mcmc/_pcn.py

```diff
@@ -17,6 +17,10 @@ def __init__(self, target, scale=1.0, **kwargs):
 
         self._acc = [1] # TODO. Check if we need this
 
+        # parameters for tune method
+        self.lambd = self.scale
```

Comment:
Please state what the parameters lambd and star_acc are used for in the tuning. Why is the value set to exactly 0.44?

Reply:
- Added references for the algorithm used:
Algorithm 4 in `Andrieu, Christophe, and Johannes Thoms. 
        "A tutorial on adaptive MCMC." Statistics and computing 18 (2008): 343-373.`
- Regarding the target acceptance rate, the original pCN paper uses 0.234, while `cuqi/sampler/pCN` uses 0.44. We consulted Felipe on this issue, he recommended taking it back to as in the original paper.
- For now, we keep it as 0.44 for comparison purposes.